docs/shared/monorepo-ci-github-actions.md

```diff
@@ -89,9 +89,6 @@ env:
   NX_CLOUD_DISTRIBUTED_EXECUTION: true
   NX_CLOUD_DISTRIBUTED_EXECUTION_AGENT_COUNT: 3
   NX_BRANCH: ${{ github.event.number || github.ref_name }}
```

Comment:
```suggestion
NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
```

we want to encourage people to declare secret read-write tokens in CI, so I think we'll need to keep the access token but we can remove everything else, since nx-cloud handles nx_branch and distributed_execution_agent_count now, and `NX_CLOUD_DISTRIBUTED_EXECUTION: true` is set on the `start-ci-run` command